mapper.go

```diff
@@ -24,7 +24,7 @@ import (
 )
 
 var (
-	metricLineRE      = regexp.MustCompile(`^(\*\.|[^*.]+\.)*(\*|[^*.]+)$`)
+	metricLineRE      = regexp.MustCompile(`^(\*\.|[^*.]+\.|\.)*(\*|[^*.]+)$`)
```

Comment:
Metric names that come from Graphite may be prefixed with 1 or 2 dots. For instance,
.a.*
..a.*
These dots are a result of unspecified group name and type for the metric in Graphite.

Just like we are trimming whitespaces in the current exporter code, we also need to trim these dots at the beginning of the string such that the mapping does not fail.